larpandoracontent/LArHelpers/LArMCParticleHelper.cc

```diff
@@ -479,7 +479,14 @@ void LArMCParticleHelper::SelectReconstructableMCParticles(const MCParticleList
 
     // Obtain vector: target mc particles
     MCParticleVector targetMCVector;
-    parameters.m_foldBackHierarchy ? LArMCParticleHelper::GetPrimaryMCParticleList(pMCParticleList, targetMCVector) : (void) std::copy(pMCParticleList->begin(), pMCParticleList->end(), std::back_inserter(targetMCVector));
+    if (parameters.m_foldBackHierarchy)
+    {
+        LArMCParticleHelper::GetPrimaryMCParticleList(pMCParticleList, targetMCVector);
+    }
+    else
+    {
+        (void) std::copy(pMCParticleList->begin(), pMCParticleList->end(), std::back_inserter(targetMCVector));
```

Comment:
No need for the preceding (void) here

Reply:
(this has been fixed in the 'indentation fix' commit)